metrics-jersey2/src/test/java/com/codahale/metrics/jersey2/SingletonMetricsResponseMeteredPerClassJerseyTest.java

```diff
@@ -126,14 +126,21 @@ public void responseMeteredUnmappedExceptionPerClassMethodsAreMetered() {
 
     @Test
     public void subresourcesFromLocatorsRegisterMetrics() {
+        final Meter meter2xx = registry.meter(name(InstrumentedSubResourceResponseMeteredPerClass.class,
+                "responseMeteredPerClass",
+                "2xx-responses"));
+        final Meter meter201 = registry.meter(name(InstrumentedSubResourceResponseMeteredPerClass.class,
```

Comment:
question: Should the variable name be `meter200` or the name suffix be `201-responses`?

Given the test class, I would expect the variable name to be different.

Reply:
Good catch! Fixed!